airqo-platform/AirQo-frontend
# Loadingindicator

#### Summary of Changes (What does this PR do?)

- changed the cachedNetworkImage provider to just cachedNetworkImage in the HealthTipContainer class of  the customwidgets.dart  file in order to add new parameters for the error widget and the container loading Animation widget which is supposed to show when as the image is loading or being fetched.

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
run the app and then when you choose any location at the part for more insights ,scroll down to today's health tips and then check if the loading animation works atleast once before the image is loaded


#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
